tests/uninstall-rgbds.bats

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bats
+
+load _helper
+
+setup_file() {
+	setup_file_common
+	setup_file_version_cache
+}
+
+setup() {
+	setup_common
+	setup_make_rgbenv
+}
+
+check_use () {
```

Comment:
check version maybe could be more meaningful here?

Reply:
renamed it to `check_used_version`